main.c

```diff
@@ -456,39 +456,62 @@ static void dbus_name_lost(GDBusConnection *connection,
 	tcmu_dbg("name lost\n");
 }
 
-int load_our_module(void) {
+static int load_our_module(void)
+{
 	struct kmod_list *list = NULL, *itr;
-	int err;
 	struct kmod_ctx *ctx;
+	int ret;
 
 	ctx = kmod_new(NULL, NULL);
 	if (!ctx) {
 		tcmu_err("kmod_new() failed\n");
 		return -1;
 	}
 
-	err = kmod_module_new_from_lookup(ctx, "target_core_user", &list);
-	if (err < 0)
-		return err;
+	ret = kmod_module_new_from_lookup(ctx, "target_core_user", &list);
+	if (ret < 0) {
+		tcmu_err("kmod_module_new_from_lookup() failed to lookup alias target_core_user\n");
+		return ret;
+	}
+
+	if (!list) {
+		tcmu_err("kmod_module_new_from_lookup() failed to find module target_core_user\n");
+		return -ENOENT;
+	}
 
 	kmod_list_foreach(itr, list) {
+		int state, err;
 		struct kmod_module *mod = kmod_module_get_module(itr);
 
-		err = kmod_module_probe_insert_module (
-			mod, KMOD_PROBE_APPLY_BLACKLIST, 0, 0, 0, 0);
-
-		if (err != 0) {
-			tcmu_err("kmod_module_probe_insert_module() for %s failed\n",
-			    kmod_module_get_name(mod));
-			return -1;
-		}
+		state = kmod_module_get_initstate(mod);
+		switch (state) {
+		case KMOD_MODULE_BUILTIN:
+			tcmu_info("Module '%s' is builtin", kmod_module_get_name(mod));
```

Comment:
We use 8 space tabs like the linux kernel. I think since we have to bounce back and forth due to also supporting target_core_user, it is easier to have similar styles.